lms/templates/dashboard.html

```diff
@@ -185,8 +185,11 @@
                 related_programs = inverted_programs.get(unicode(entitlement.course_uuid if is_unfulfilled_entitlement else session_id))
                 show_consent_link = (session_id in consent_required_courses)
                 course_overview = enrollment.course_overview
+                resume_button_url = None
+                if resume_buttons_are_available:
```

Comment:
Why have this boolean while the truthy of the `resume_button_url` should be enough?

Reply:
You actually revealed a bug in my implementation. My revision fixes it. Thanks for pointing this out.